### ISSUE TITLE
shuffle and scramble are still failing

### ISSUE THREAD

**Issue by diegodorado** [tidalcycles/Tidal#463]
scramble and shuffle doesnt produce any sound.

this doesnt produce any sound
```d1 $ shuffle 2 $  s "bd hh sd hh"```

although, this does
```d1 $  s "bd hh sd hh"```

**Expected behavior**
should suffle or scramble, but does not.

**Environment (please complete the following information):**
 - OS: Linux
 - Haskell version : 8.4.2
 - Tidal version: 1.0.7
 - SuperCollider version: 3.9.1

**Comment by yaxu** [comment]
That's strange, it works for me.
If you eval `tidal_version` in a tidal buffer does it say 1.0.7?

**Comment by diegodorado** [comment]
right! I did not upgrade correctly. it is workin on tidal 1.0.7 .... although it receives Int only.... it is not patternable =(
